### PR TITLE
Run verify labels with make rules

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -66,6 +66,10 @@ update-gofmt:
 	hack/make-rules/update/gofmt.sh
 verify-gofmt:
 	hack/make-rules/verify/gofmt.sh
+update-labels:
+	hack/make-rules/update/labels.sh
+verify-labels:
+	hack/make-rules/verify/labels.sh
```
Comment (BenTheElder) [inline Makefile:72]: These should also be `.PHONY`.
https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

File: hack/make-rules/verify/all.sh [hunk 2]
```diff
@@ -35,6 +35,9 @@ if [[ "${VERIFY_GOFMT:-true}" == "true" ]]; then
   hack/make-rules/verify/gofmt.sh || res=1
   cd "${REPO_ROOT}"
 fi
+if [[ "${VERIFY_LABELS:-true}" == "true" ]]; then
+  echo "verifying labels ..."
+  hack/make-rules/verify/labels.sh || res=1
```
Comment (BenTheElder) [inline hack/make-rules/verify/all.sh:40]: missing a `fi` here.

File: Makefile (hunk 1 above)
Comment (chaodaiG) [reply Makefile]: added

File: hack/make-rules/verify/all.sh (hunk 2 above)
Comment (chaodaiG) [reply hack/make-rules/verify/all.sh]: added

**Review by fejta — APPROVED** [review]

**Pull request merged**